src/markdown-pages/create-installer/choosing-a-pv-provisioner.md

```diff
@@ -0,0 +1,75 @@
+
+```yaml
+apiVersion: "cluster.kurl.sh/v1beta1"
+kind: "Installer"
+metadata:
+  name: "local"
+spec:
+  ...
+  openebs:
+    version: "3.3.x"
+    isLocalPVEnabled: true
+    localPVStorageClassName: "default"
+```
+
+Conversely, [Rook](/docs/add-ons/rook) provides dynamic PV provisioning of distributed [Ceph](https://ceph.io/) storage. Ceph is a self-healing and self-managing, distributed storage solution. Rook volumes are accessible from any node in the cluster and are suitable for applications deployed to multi-node clusters that provide their own replication and fault tolerance (e.g. MongoDB or Cassandra), or ones where data loss is not catastrophic (e.g. a cache that can be rebuilt). Drawbacks to the Rook provisioner include slower volume performance, higher disk utilization as data is replicated across the cluster, higher CPU and memory requirements, and a minimum cluster size of 3 nodes as well as an additional, dedicated block device attached to each node. In addition to these drawbacks, Ceph adds additional complexity that should be avoided unless necessary. For more information about Rook, see the [add-on documentation](/docs/add-ons/rook).
+
+```yaml
+apiVersion: "cluster.kurl.sh/v1beta1"
+kind: "Installer"
+metadata:
+  name: "distributed"
+spec:
+  ...
+  rook:
+    version: "1.7.x"
+    storageClassName: "distributed"
+    isSharedFilesystemDisabled: true
+```
+
+In addition to providing block storage, Rook provides S3 compatible object storage. The [MinIO add-on](/docs/add-ons/minio) is offered for applications that need object storage but prefer to leverage OpenEBS Local PV.
+
+```yaml
+apiVersion: "cluster.kurl.sh/v1beta1"
+kind: "Installer"
+metadata:
+  name: "openebs-with-minio"
+spec:
+  ...
+  openebs:
+    version: "3.3.x"
+    isLocalPVEnabled: true
+    localPVStorageClassName: "default"
+  minio:
+    version: "2022-09-07T22-25-02Z"
+```
+
+It is recommended to evaluate each individual workload's need for data persistence and choose the provisioner that is most suitable. In many cases it will be desirable to deploy both provisioners. In this configuration, Rook will be the default storage provisioner for PersistentVolumeClaims that do not specify a `storageClassName`.
```

Comment:
```suggestion
It is recommended to evaluate each individual workload's need for data persistence and choose the provisioner that is most suitable.
In many cases it will be desirable to deploy both provisioners.
In this configuration, Rook will be the default storage provisioner for PersistentVolumeClaims that do not specify a `storageClassName`.
```

Reply:
I do not believe this is currently true - IIRC if you have openebs with a storageclass 'local' and rook with a storageclass 'distributed' neither will be default.

Unless we changed that logic recently?